### DISCUSSION TITLE
po-menu (comportamento mouseEnter e mouseLeave)

### DISCUSSION THREAD

**Discussion by rochagc** [po-ui/po-angular#1888] [Tire suas dúvidas] [answered]
Olá, estou utilizando a versão **16.6.0** do po-ui e existe um comportamento do componente **po-menu**, que se expande ou colapsa automaticamente quando o mouse passa por cima. Gostaria de saber se existe uma forma de suprimir esse comportamento?
Preciso que o menu se expanda apenas quando eu clico no mesmo e, logo após clicar em um item, o menu colapse novamente. Esse comportamento existia em versões bem mais antigas do po-ui (5.22.1). Sei que é uma versão muito antiga, mas após migrarmos dela para a 16.6.0, notamos essa mudança no comportamento do menu e nosso PO gostaria que o comportamento antigo permanecesse.
Existe alguma alternativa?
Obrigado.

https://github.com/po-ui/po-angular/assets/40344086/b6c17471-1494-4eb0-8298-9c60a173283e

**Comment by anliben** [comment, marked as answer]
Boa Tarde @rochagc, tudo bem?

Iremos responder sobre este item no jira!

Attr.
